k8s/nginx/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,36 @@
 actions:
```

Comment:
It's not a valid test file for testrunner. Please rename it by adding `.template` suffix.

Reply:
Done.